src/notation/internal/notationactioncontroller.cpp

```diff
@@ -1729,6 +1729,17 @@ Ms::EngravingItem* NotationActionController::selectedElement() const
     return selection ? selection->element() : nullptr;
 }
 
+bool NotationActionController::noteOrRestSelected() const
+{
+    EngravingItem* element = selectedElement();
```

Comment:
Just copy the logic from resolveTupletEnabled
Your code will not work for range selection

Reply:
Good spot, @Eism!